llvm/test/CodeGen/AMDGPU/divergent-branch-uniform-condition.ll

```diff
@@ -5,8 +5,8 @@
 ;       cannot be expressed reliably by the generated checks.
 
 ; RUN: llc -mtriple=amdgcn -mcpu=gfx900 < %s | FileCheck %s -check-prefix=ISA
-; RUN: opt --amdgpu-annotate-uniform -S %s |  FileCheck %s -check-prefix=UNIFORM
-; RUN: opt --amdgpu-annotate-uniform --si-annotate-control-flow -S %s |  FileCheck %s -check-prefix=CONTROLFLOW
+; RUN: opt -mtriple=amdgcn-mesa-mesa3d --amdgpu-annotate-uniform -S %s |  FileCheck %s -check-prefix=UNIFORM
+; RUN: opt -mtriple=amdgcn-mesa-mesa3d --amdgpu-annotate-uniform --si-annotate-control-flow -S %s |  FileCheck %s -check-prefix=CONTROLFLOW
```

Comment:
Just drop the changes to this file. It already has a target triple. If you want to move it to the command line, remove the in-file field

Reply:
The opt invocation doesn't have the triple. The UA needs the target detail to compute the uniformity data.